base/ca/src/com/netscape/ca/CRLIssuingPoint.java

```diff
@@ -1767,8 +1783,33 @@ private long findNextUpdate(boolean fromLastUpdate, boolean delta) {
             }
         }
 
+        logger.debug("CRLIssuingPoint.findNextUpdate: nextUpdate : " + new Date(nextUpdate) + " next: " + new Date(next));
+
         if (fromLastUpdate && nextUpdate > 0 && (nextUpdate < next || nextUpdate >= now)) {
-            next = nextUpdate;
+            // We have the one time schedule updated flag set in CS.cfg, which means
+            // we want the schedule adhered to now instead of waiting for the next update for it
+            // to take effect.
+            // Here the variable "next" has the newly calculated nextUpdate value
+            // Here the variable "nextUpdate" contains the nextUpdate value from the previous schedule
+            if (mScheduleUpdated) {
```

Comment:
I'd probably just simply this and set it to update now regardless of the value, and start off the new interval fresh.

Reply:
I think I will roll with the current interpretation so the user will know that their new schedule will attempt to be adjusted for.